addons/event/__manifest__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 {
     'name': 'Events Organization',
-    'version': '1.8',
+    'version': '1.9',
```

Comment:
It was already bumped for 16.5, no need to re-bump :) (existing scripts for 1.7 and 1.8 already for 16.5 :D )

Reply:
ok :) same for event_sale I suppose then ?